src/Workspaces/MSBuild/Test/VisualStudioMSBuildWorkspaceTests.cs

```diff
@@ -3285,7 +3285,7 @@ public async Task TestInvalidSolutionFilterDoesNotLoad()
             var solutionFilePath = GetSolutionFileName(@"InvalidSolutionFilter.slnf");
 
             using var workspace = CreateMSBuildWorkspace();
-            var exception = await Assert.ThrowsAsync<Exception>(() => workspace.OpenSolutionAsync(solutionFilePath));
+            var exception = await Assert.ThrowsAsync<RemoteInvocationException>(() => workspace.OpenSolutionAsync(solutionFilePath));
```

Comment:
Is this the RPC layer's RemoteInvocationException? I wouldn't imagine we should be throwing this directly, or if we are we shouldn't be asserting that's what's being thrown. Should the test be changed to ThrowsAnyAsync?

Reply:
Will do!